parquet-generator/src/main/java/org/apache/parquet/encoding/bitpacking/ByteBasedBitPackingGenerator.java

```diff
@@ -256,4 +254,8 @@ private static String align(int value, int digits) {
     result.append(valueString);
     return result.toString();
   }
+  
+  private static int paddedByteCountFromBits(int bitLength) {
```

Comment:
I had to break circular dependency between parquet-generator and parquet-common. 

This was the only method from parquet-common that was used by parquet-generator, so I've just copied it here and removed dependency on parquet-common (have a look at parquet-generator/pom.xml change).